test/test_helper.exs

```diff
@@ -3,5 +3,3 @@ Application.ensure_all_started(:mox)
 Application.ensure_all_started(:stream_data)
 
 Mox.defmock(ExHal.ClientMock, for: ExHal.Client)
-
-Application.put_env(:exhal, :client, ExHal.ClientMock)
```

Comment:
Removed from test_helper.exs since not all tests wish to mock using Mox. Moving Navigation to use runtime client_module this causes issues in testing.